build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "org.jetbrains.research.ml.pluginUtilities"
+version = "1.0"
+
+fun properties(key: String) = project.findProperty(key).toString()
+
+plugins {
+    java
+    kotlin("jvm") version "1.5.10" apply true
+    id("org.jetbrains.intellij") version "1.0" apply true
+    id("org.jetbrains.dokka") version "1.4.30" apply true
+    id("org.jlleitschuh.gradle.ktlint") version "10.0.0" apply true
+}
+
+allprojects {
+    apply {
+        plugin("java")
+        plugin("kotlin")
+        plugin("org.jetbrains.intellij")
+        plugin("org.jetbrains.dokka")
+        plugin("org.jlleitschuh.gradle.ktlint")
+    }
+
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        implementation(kotlin("stdlib-jdk8"))
```

Comment:
Don't hate me for that version definition review :) But you set JDK 11, why do you specify JDK-8 here?